src/app/services/app.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable, Provider } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+
+@Injectable()
+export class AppService {
```

Comment:
I think `TimeService` would be more appropriate.

Reply:
done